src/main/java/net/snowflake/client/core/SFBaseStatement.java

```diff
@@ -191,7 +191,7 @@ public static boolean isFileTransfer(String sql) {
    * Retrieves the current result as a ResultSet, if any. This is invoked by SnowflakeStatement and
    * should return an SFBaseResultSet, which is then wrapped in a SnowflakeResultSet.
    *
-   * @return SFBaseResultSet
+   * @return SFBaseResultSet {@link SFBaseResultSet}
```

Comment:
it could be just `* @return {@link SFBaseResultSet}` wherever the link is used, let's not double the text